src/Common/src/Interop/OSX/System.Native/Interop.ProtocolStatistics.cs

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Sys
+    {
+        [StructLayoutAttribute(LayoutKind.Sequential)]
+        public unsafe struct TcpGlobalStatistics
+        {
+            public readonly ulong ConnectionsAccepted;
+            public readonly ulong ConnectionsInitiated;
+            public readonly ulong CumulativeConnections;
+            public readonly ulong ErrorsReceived;
+            public readonly ulong FailedConnectionAttempts;
+            public readonly ulong SegmentsReceived;
+            public readonly ulong SegmentsResent;
+            public readonly ulong SegmentsSent;
+            public readonly int CurrentConnections;
+            private readonly int __padding;
```

Comment:
@pgavlin answered my question below. The naming is not following our naming convention but I would leave this way if other shim code uses the same convention for this special member.